docs/plugins/jenkins.md

```diff
@@ -4,15 +4,40 @@ This plugin installs [jenkins](https://jenkins.io) in an existing Kubernetes clu
 
 ## Usage
 
-NOTICE:
+### NOTICE
 
-Create the data directory manually:
+#### Production Environments
```

Comment:
Level 4 titles should reside under level 3 titles. Please adjust the title levels and doc structure. Same for below titles.